daemon.go

```diff
@@ -169,8 +170,10 @@ func (d *daemon) runCheck(query url.Values) (*output, error) {
 
 	if !connectionFailed {
 		// Is the target connectable
-		dialCtx, dialCancel := context.WithTimeout(ctx, time.Second*3)
+		dialCtx, dialCancel := context.WithTimeout(ctx, time.Second*6)
 		connErr := testHost.Connect(dialCtx, *ai)
+		// Testing shows that it's pretty common for Connect to succeeed but for NewStream to fail.
```

Comment:
clarification: where/what is `NewStream` ?

Reply:
It's in Vole's Bitswap Check
https://github.com/ipfs-shipyard/vole/commit/d7b80a4611935295799d439967786f3be699e25a#diff-ae002c00aee78527011bc652c7ac400e9f3f25f753652aaa237562a4ed66b2d1R81-R85

Reply:
every protocol typically uses its own stream. The point here is that Connect can succeed connecting to a peer with a circuit relay, but NewStream will actually cause the hole punching to happen.